src/app/pages/courses-page/courses-page.component.html

```diff
@@ -1 +1,7 @@
-<p>courses-page works!</p>
+<app-breadcrumbs></app-breadcrumbs>
```

Comment:
this must be placed on app's layout level

Reply:
Will do